src/Checks/MigrationUpToDateHealthCheck.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace UKFast\HealthCheck\Checks;
+
+use Illuminate\Database\Migrations\Migrator;
+use UKFast\HealthCheck\HealthCheck;
+use UKFast\HealthCheck\Status;
+
+class MigrationUpToDateHealthCheck extends HealthCheck
+{
+    protected $name = 'migration';
+
+    /**
+     * @var Migrator
+     */
+    protected $migrator;
+
+    /**
+     * @return Status
+     */
+    public function status()
+    {
+        try {
+            if (!$this->isDatabaseUpToDate()) {
+                return $this->problem('Not all migrations have been executed', []);
```

Comment:
I think it'd be nice to return the migrations which have not been ran in the problem context here.

Reply:
@tylerwoonton thx for your usefull feedback, i will add that and update the pr